CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## 6.1.0 (Unreleased)
+#### Notes
+Extends support of the SDK to OneView REST API version 2800 (OneView v6.10) and ImageStreamer REST API version 2020 (I3S v6.10).
+
+#### Features supported
+- Artifact Bundle
+- Build Plan
+- Connection Template
+- Deployment Group
+- Deployment Plan
+- Enclosure
+- Enclosure Group
+- Ethernet Network
+- FC Network
+- FCOE Network
+- Firmware Driver
+- Golden Image
+- Hypervisor Cluster Profile
+- Hypervisor Manager
+- Interconnects
+- Interconnect Type
+- LIG Uplink Set
+- Logical Enclosure
+- Logical Interconnect
+- Logical Interconnect Group
+- Network Set
+- OS Deployment Plan
```

Comment:
Add lib and spec files for os_deployment_plan as it is being used by HCP.

Reply:
fixed